nxdrive/data/i18n/i18n-he.json

```diff
@@ -257,7 +257,7 @@
     "SELECT_SYNC_FOLDERS": "Choose folders to sync",
     "SELECT_SYNC_FOLDERS_DESCRIPTION": "Allows you to choose which folders to sync locally. This requires the engine to be online.",
     "SERVER_DEFAULT": "Server default",
-    "SERVER_INCOMPATIBLE": "Your server version is not compatible with Drive %1. We advise you to downgrade your Drive client to an older version or to contact your administrator to upgrade your Nuxeo instance.",
```

Comment:
Such fixes have to be done from Crowdin. I will explain how it works in the afternoon.